app/models/user.js

```diff
@@ -44,8 +44,15 @@ module.exports = (sequelize, DataTypes) => {
       }
     }).catch(error => databaseError(error.message));
 
-  User.findUser = email => User.findOne({ where: { email } }).catch(error => databaseError(error.message));
+  User.findUserByEmail = email =>
+    User.findOne({ where: { email } }).catch(error => databaseError(error.message));
 
+  User.getUserPassword = user => {
+    if (user) {
+      return user.password;
+    }
+    throw validationError("user must be valid. It can't be null");
+  };
```

Comment:
What I meant in my previous review is that this function is not really necessary, and the empty validation should go in the controller or in the `findUserByEmail ` function. Mostly because the empty check is generic but you are only obtaining the password. Another option is to use a generic null check function and then obtain whatever you need via user.password or user.name or whatever.